hw/hw17/html_render.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+
+"""
+Python class example.
+
+"""
+
+# The start of it all:
+# Fill it all in here.
+
+
+class Element(object):
+
+    def __init__(self, name="", content=""):
+        self.name = name
+        # self.content = content
+        self.children = [content] if content else []
+
+    def append(self, new_child):
+            self.children.append(new_child)
+
+    def render(self, file_out, ind="    "):
+        self.indent = ind
+        IND_LEVEL = "    "
+        # append the html thingies, indent, and content and write file
+        file_out.write("%s<%s>" % (self.indent, self.name))
+        for child in self.children:
```

Comment:
Good though I think you have an issue when you render elements inside of elements. In the second output file, you have two tags on the same line, so you want to make sure you're starting a new line.